model/src/main/scala/org/broadinstitute/dsde/rawls/StringValidationUtils.scala

```diff
@@ -10,79 +10,97 @@ trait StringValidationUtils {
   implicit val errorReportSource: ErrorReportSource
 
   private lazy val userDefinedRegex = "[A-z0-9_-]+".r
-  def validateUserDefinedString(s: String): Unit = {
-    if(! userDefinedRegex.pattern.matcher(s).matches) {
+  def validateUserDefinedString(s: String): Unit =
+    if (!userDefinedRegex.pattern.matcher(s).matches) {
       val msg = s"Invalid input: $s. Input may only contain alphanumeric characters, underscores, and dashes."
-      throw new RawlsExceptionWithErrorReport(errorReport = ErrorReport(message = msg, statusCode = StatusCodes.BadRequest))
+      throw new RawlsExceptionWithErrorReport(
+        errorReport = ErrorReport(message = msg, statusCode = StatusCodes.BadRequest)
+      )
     }
-  }
 
   private lazy val entityNameRegex = "[A-z0-9\\._-]+".r
-  def validateEntityName(s: String): Unit = {
-    if(! entityNameRegex.pattern.matcher(s).matches) {
-      val msg = s"Invalid input: $s. Input may only contain alphanumeric characters, underscores, dashes, and periods."
-      throw new RawlsExceptionWithErrorReport(errorReport = ErrorReport(message = msg, statusCode = StatusCodes.BadRequest))
+  def validateEntityName(s: String): Unit =
+    if (!entityNameRegex.pattern.matcher(s).matches) {
+      val msg =
+        s"Invalid entity name: $s. Input may only contain alphanumeric characters, underscores, dashes, and periods. If importing a snapshot, making an API call, uploading a TSV, or adding a new row from the Terra UI, please modify the impacted name(s) and try again."
```

Comment:
you don't have to hold the PR for this, but for the future: use three double quotes to support multiline strings for readability, like:
```scala
      val msg =
        s"""Invalid entity name: $s. Input may only contain alphanumeric characters, underscores, dashes, and periods. 
           |If importing a snapshot, making an API call, uploading a TSV, or adding a new row from the Terra UI, 
           |please modify the impacted name(s) and try again.""".stripMargin
```

Reply:
Thank you!  That was bothering me.  I'm gonna fix it.